test/rulesets/Base/arraymath.jl

```diff
@@ -49,6 +49,43 @@
     end
 
 
+    VERSION >= v"1.6.0-DEV.1536" && @testset "muladd: $T" for T in (Float64, ComplexF64)
```

Comment:
Arn't there some versions of this that work pre v1.6?
Shouldn't we should test those and then just `continue` in the ones that don't

Reply:
Maybe. 

Although whether it's worth providing gradients for `x*y+z` is another question. Possibly these definitions shouldn't be loaded at all on pre-1.6 Julia. 

The other option would be to load Compat.jl, and then always have them. Does ChainRules have strong feelings about this?

Reply:
ChainRules already loads Compat.

Reply:
OK, then that's simple.

Reply:
This is now resolved